packages/css-engine/src/core/style-element.ts

```diff
@@ -1,12 +1,16 @@
 export class StyleElement {
   #element?: HTMLStyleElement;
+  #name: string;
+  constructor(name = "") {
+    this.#name = name;
+  }
   get isMounted() {
     return this.#element?.parentElement != null;
   }
   mount() {
     if (this.isMounted === false) {
       this.#element = document.createElement("style");
-      this.#element.setAttribute("data-webstudio", "");
+      this.#element.setAttribute("data-webstudio", this.#name);
```

Comment:
Why not something like data-ws-style similar to data-ws-id and data-ws-component?

Reply:
I remember we had a conversation that "ws" isn't meaningful and confusing with websockets so I called it webstudio and thought we should use webstudio full name everywhere 🤷‍♂️ but didn't make it consistent, I guess since this question is now here, we should make a consistent naming